app/data/contentscript.js

```diff
@@ -3,31 +3,31 @@
 
 var pathComponents = window.location.pathname.split('/');
 var _logName = 'github-forks-extension:';
-var lovelyForksClass = 'lovely-forks-addon-class';
 var DEBUG = false;
+var el = new WeakMap();
```

Comment:
But… I guess it is an overkill. It's just an element and it's likely to simply be replaced once the ajax navigation occurs. I can fix.